.docker-env

```diff
@@ -7,5 +7,5 @@ PGDATABASE=nmdc_a
 NMDC_DATABASE_URI="postgresql://postgres:postgres@db/nmdc_a"
 NMDC_CELERY_BACKEND="redis://redis:6379/0"
 NMDC_CELERY_BROKER="redis://redis:6379/0"
-NMDC_INGEST_DATABASE_URI="postgresql://postgres:postgres@db/nmdc_a"
+NMDC_INGEST_DATABASE_URI="postgresql://postgres:postgres@db/nmdc_b"
```

Comment:
I think this change is technically unnecessary; **but** I do like it because it implies that someone expects `NMDC_DATABASE_URI` and `NMDC_INGEST_DATABASE_URI` to have different values (and I, for example, do expect that).